backend/src/routes/api/scene.js

```diff
@@ -27,10 +29,27 @@ router.get("/full/:sceneId", async (req, res) => {
 // Retrieve all scenes of a scenario
 router.get("/", async (req, res) => {
   const scenes = await retrieveSceneList(req.params.scenarioId);
-
   res.json(scenes);
 });
 
+// Get graph
+router.get("/graph", async (req, res) => {
+  const scenes = await retrieveSceneList(req.params.scenarioId);
+  let fullScenes = [];
+  
+  // Get FULL scene data - should really change this to run in parallel for better runtime
+  for await (const scene of scenes) {
+    const fullScene = await retrieveScene(scene._id);
+    fullScenes.push(fullScene);
+  }
+
+  const scenarioGraph = new ScenarioGraph(fullScenes);
+  const distance = scenarioGraph.distanceBetween("6295a71dc02aa66b701339b2", "62f4805867e5e917c0ef095c");
+  console.log(distance);
```

Comment:
I assume this is for testing? 

Maybe now a good time to chat with @Harshal-D and @nickykay98 about how this endpoint should be used, and this will link into the idea of "bad ends" and "successful end nodes". This will make the purpose of this endpoint a lot clearer.